app/jobs/kit_import_job.rb

```diff
@@ -16,22 +17,27 @@ def perform(file_or_folder, logger:, user_id: nil)
     @current_user = user_id ? User.find(user_id) : User.first
     @logger = logger
     @project = nil
-    @report_templates_dir = Configuration.paths_templates_reports
+    @templates_dirs = TEMPLATE_TYPES.map do |template_type|
+      [
+        template_type,
+        Pathname.new(Configuration.send("paths_templates_#{template_type}"))
```

Comment:
`paths_templates_#{template_type}` doesn't because we call [to_s](https://github.com/dradis/dradis-ce/blob/5ed0b25bc2149228b2ff77c515ed42a01075de4e/app/models/configuration.rb#L52)

Reply:
If i build the path using `paths_templates` it may not be accurate because they could be using a different path for any specific configuration